BepressImportDom.inc.php

```diff
@@ -167,13 +170,13 @@ function _handleArticleNode() {
 		$this->_dependentItems[] = 'article';
 
 		// Create publication object and add info
+		/** @var PublicationDAO $publicationDao */
 		$publicationDao = DAORegistry::getDAO('PublicationDAO');
 
 		$publication = $publicationDao->newDataObject();
 		/** @var $publication PKPPublication */
 		$publication->setData('submissionId', $this->_submission->getId());
-		$publication->setData('locale', $this->_primaryLocale);
-		$publication->setData('languages', 'en');
+		$publication->setData('language', PKPLocale::getIso1FromLocale($this->_primaryLocale));
```

Comment:
Shouldn't be `languages` and an array as value?
Reference: https://github.com/pkp/pkp-lib/blob/stable-3_3_0/classes/publication/PKPPublicationDAO.inc.php#L73